src/pages/test-results-search/components/search-result/search-result.html

```diff
@@ -18,12 +18,12 @@ <h3>{{getName()}}</h3>
       <ion-col col-8 no-padding>
         <h2>{{searchResult.category}}</h2>
       </ion-col>
-      <ion-col col-16 no-padding>
-        <h2>{{searchResult.activityCode}}</h2>
+      <ion-col col-16 no-padding padding-right>
```

Comment:
no-padding and padding-right?

Reply:
It's something we do on all the test slots removes the left , top and bottom padding but keeps thr right